foundations/foundation-common/src/main/java/io/servicecomb/foundation/common/utils/RSAKeyPairEntry.java

```diff
@@ -0,0 +1,33 @@
+package io.servicecomb.foundation.common.utils;
```

Comment:
code template is not correct, please check etc folder for eclipse and IDEA code template

Reply:
accept , forget to format